### PR TITLE
Fix superres voxel shapenet

### PR DESCRIPTION
**Pull request by Jean-Francois-Lafleche**
Minimal changes to get example functioning.

- Fixed syntax to comply with Flake8.
- Condensed MSE and NLLL examples to reduce duplicate code.
- Tested with new ShapeNet dataset changes.

**Tests**
OS: Ubuntu 18.04
CUDA: 10.1
GPU: RTX 2080Ti
Driver: 435.21

Tests performed:
For each training run, ensured that 5 epochs were successfully trained.
- Train MSE:  `python examples/SuperResolution/voxel-ShapeNet/train.py --shapenet-root data/ShapeNet/ --categories bowl --cache_dir cache/ --loss-type MSE`
- Train NLLL: `python examples/SuperResolution/voxel-ShapeNet/train.py --shapenet-root data/ShapeNet/ --categories bowl --cache_dir cache/ --loss-type NLLL`
- evaluate MSE: `python examples/SuperResolution/voxel-ShapeNet/eval.py --shapenet-root data/ShapeNet/ --categories bowl --cache_dir cache/ --loss-type MSE`
- evaluate NLLL: `python examples/SuperResolution/voxel-ShapeNet/eval.py --shapenet-root data/ShapeNet/ --categories bowl --cache_dir cache/ --loss-type NLLL`

### CODE REVIEW

File: examples/SuperResolution/voxel-ShapeNet/eval.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+import kaolin as kal
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--shapenet-root', type=str, required=True, help='Root directory of the ShapeNet dataset.')
+parser.add_argument('--cache_dir', type=str, default='cache', help='Root directory of the ShapeNet dataset.')
+parser.add_argument('--loss-type', type=str, choices=['MSE', 'NLLL'], default='MSE', help='Specify the loss type to use.')
+parser.add_argument('--device', type=str, default='cuda', help='Device to use')
+parser.add_argument('--categories', type=str, nargs='+', default=['chair'], help='list of object classes to use')
+parser.add_argument('--vis', action='store_true', help='Visualize each model while evaluating')
+parser.add_argument('-batchsize', type=int, default=16, help='Batch size.')
```
Comment (Caenorst) [inline examples/SuperResolution/voxel-ShapeNet/eval.py]: I think it should be `--batchsize` instead of `-batchsize`

File: examples/SuperResolution/voxel-ShapeNet/train.py [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+
+parser = argparse.ArgumentParser()
+parser.add_argument('--shapenet-root', type=str, required=True, help='Root directory of the ShapeNet dataset.')
+parser.add_argument('--cache_dir', type=str, default='cache', help='Root directory of the ShapeNet dataset.')
+parser.add_argument('--loss-type', type=str, choices=['MSE', 'NLLL'], default='MSE', help='Specify the loss type to use.')
+parser.add_argument('--device', type=str, default='cuda', help='Device to use')
+parser.add_argument('--categories', type=str, nargs='+', default=['chair'], help='list of object classes to use.')
+parser.add_argument('--epochs', type=int, default=30, help='Number of train epochs.')
+parser.add_argument('--batchsize', type=int, default=16, help='Batch size.')
+parser.add_argument('-lr', '--learning_rate', type=float, default=1e-3, help='Learning rate.')
```
Comment (Caenorst) [inline examples/SuperResolution/voxel-ShapeNet/train.py:26]: like comment above: `--lr`

**Review by Caenorst — COMMENTED** [review]
Great work ! Few little "typos" to fix and we are good to go. Please specify the tests that you ran and under what platforms.

File: examples/SuperResolution/voxel-ShapeNet/train.py (hunk 2 above)
Comment (Jean-Francois-Lafleche) [reply examples/SuperResolution/voxel-ShapeNet/train.py:26]: You're totally right. Fixed one but not the other.

Comment (Caenorst) [reply examples/SuperResolution/voxel-ShapeNet/train.py:26]: Actually it's better this way, didn't see that there were the option '--learning_rate'

**Review by Caenorst — APPROVED** [review]
LGTM, thanks for the PR !
